The job **`x86_64-gnu-llvm-19`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/14610555702/job/40987711445) [(plain)](https://github.com/rust-lang/rust/commit/b44e43cc5a4a5474114248249bb349d2c445b7e3/checks/40987711445/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
#20 exporting to docker image format
#20 sending tarball 19.7s done
#20 DONE 25.7s
##[endgroup]
Setting extra environment values for docker:  --env ENABLE_GCC_CODEGEN=1 --env GCC_EXEC_PREFIX=/usr/lib/gcc/
[CI_JOB_NAME=x86_64-gnu-llvm-19]
[CI_JOB_NAME=x86_64-gnu-llvm-19]
debug: `DISABLE_CI_RUSTC_IF_INCOMPATIBLE` configured.
---
sccache: Listening on address 127.0.0.1:4226
##[group]Configure the build
configure: processing command line
configure: 
configure: build.configure-args := ['--build=x86_64-unknown-linux-gnu', '--llvm-root=/usr/lib/llvm-19', '--enable-llvm-link-shared', '--set', 'rust.randomize-layout=true', '--set', 'rust.thin-lto-import-instr-limit=10', '--set', 'build.print-step-timings', '--enable-verbose-tests', '--set', 'build.metrics', '--enable-verbose-configure', '--enable-sccache', '--disable-manage-submodules', '--enable-locked-deps', '--enable-cargo-native-static', '--set', 'rust.codegen-units-std=1', '--set', 'dist.compression-profile=balanced', '--dist-compression-formats=xz', '--set', 'rust.lld=false', '--disable-dist-src', '--release-channel=nightly', '--enable-debug-assertions', '--enable-overflow-checks', '--enable-llvm-assertions', '--set', 'rust.verify-llvm-ir', '--set', 'rust.codegen-backends=llvm,cranelift,gcc', '--set', 'llvm.static-libstdcpp', '--enable-new-symbol-mangling']
configure: build.build          := x86_64-unknown-linux-gnu
configure: target.x86_64-unknown-linux-gnu.llvm-config := /usr/lib/llvm-19/bin/llvm-config
configure: llvm.link-shared     := True
configure: rust.randomize-layout := True
configure: rust.thin-lto-import-instr-limit := 10
---
  Number of decisions:   4447
  longest path:          1159 (code:    152)
  longest backtrack:       66 (code:    428)
Shared 86733 out of 152951 states by creating 14756 new states, saving 71977
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/expmed.cc: In function ‘rtx_def* extract_bit_field_1(rtx, poly_uint64, poly_uint64, int, rtx, machine_mode, machine_mode, bool, bool, rtx_def**)’:
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/expmed.cc:1864:45: warning: ‘*(unsigned int*)((char*)&imode + offsetof(scalar_int_mode, scalar_int_mode::m_mode))’ may be used uninitialized [-Wmaybe-uninitialized]
 1864 |       rtx sub = extract_bit_field_as_subreg (mode1, op0, imode,
      |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~
 1865 |                                              bitsize, bitnum);
      |                                              ~~~~~~~~~~~~~~~~
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/expmed.cc:1824:19: note: ‘*(unsigned int*)((char*)&imode + offsetof(scalar_int_mode, scalar_int_mode::m_mode))’ was declared here
 1824 |   scalar_int_mode imode;
      |                   ^~~~~
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/gimple-range-gori.cc: In member function ‘void range_def_chain::dump(FILE*, basic_block, const char*)’:
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/gimple-range-gori.cc:319:19: warning: format not a string literal and no format arguments [-Wformat-security]
  319 |           fprintf (f, prefix);
      |           ~~~~~~~~^~~~~~~~~~~
---
                 from /checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/analyzer/region-model.h:33,
                 from /checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/analyzer/access-diagram.cc:39:
In constructor ‘ana::byte_range::byte_range(ana::byte_offset_t, ana::byte_size_t)’,
    inlined from ‘virtual text_art::table ana::string_literal_spatial_item::make_table(const ana::bit_to_table_map&, text_art::style_manager&) const’ at /checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/analyzer/access-diagram.cc:1812:18:
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/analyzer/store.h:312:5: warning: ‘size_in_bytes.generic_wide_int<fixed_wide_int_storage<128> >::fixed_wide_int_storage<128>.fixed_wide_int_storage<128>::val[1]’ may be used uninitialized [-Wmaybe-uninitialized]
  312 |     m_size_in_bytes (size_in_bytes)
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/analyzer/access-diagram.cc: In member function ‘virtual text_art::table ana::string_literal_spatial_item::make_table(const ana::bit_to_table_map&, text_art::style_manager&) const’:
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/analyzer/access-diagram.cc:1808:28: note: ‘size_in_bytes.generic_wide_int<fixed_wide_int_storage<128> >::fixed_wide_int_storage<128>.fixed_wide_int_storage<128>::val[1]’ was declared here
 1808 |                byte_size_t size_in_bytes
      |                            ^~~~~~~~~~~~~
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/diagnostic.cc: In function ‘void fancy_abort(const char*, int, const char*)’:
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/diagnostic.cc:1677:15: warning: format not a string literal and no format arguments [-Wformat-security]
 1677 |       fnotice (stderr, diagnostic_kind_text[DK_ICE]);
---
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/gcc.cc:7930:9: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
 7930 |   write (fd, "\n\n", 2);
      |   ~~~~~~^~~~~~~~~~~~~~~
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/gcc.cc: In member function ‘void driver::final_actions() const’:
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/gcc.cc:9307:13: warning: ignoring return value of ‘int truncate(const char*, __off_t)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
 9307 |     truncate(totruncate_file, 0);
      |     ~~~~~~~~^~~~~~~~~~~~~~~~~~~~
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/gengtype-lex.l: In function ‘int yylex(const char**)’:
gengtype-lex.cc:357:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
  357 | #define YY_DO_BEFORE_ACTION \
      |  ~~~~~~~~~~~~~^~~~~~~
---
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/lto/lto-common.cc: In function ‘void lto_resolution_read(splay_tree, FILE*, lto_file*)’:
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/lto/lto-common.cc:2091:10: warning: ignoring return value of ‘int fscanf(FILE*, const char*, ...)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
 2091 |   fscanf (resolution, " ");   /* Read white space.  */
      |   ~~~~~~~^~~~~~~~~~~~~~~~~
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/lto/lto-common.cc:2093:9: warning: ignoring return value of ‘size_t fread(void*, size_t, size_t, FILE*)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
 2093 |   fread (obj_name, sizeof (char), name_len, resolution);
      |   ~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/lto/lto-common.cc:2113:10: warning: ignoring return value of ‘int fscanf(FILE*, const char*, ...)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
 2113 |   fscanf (resolution, "%u", &num_symbols);
      |   ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/gcc/jit/jit-recording.cc:32:
---
Applying io_quotes_use            to linux/blkzoned.h
Applying io_quotes_use            to linux/ipmi.h
Applying io_quotes_use            to linux/psp-dbc.h
Applying io_quotes_use            to linux/bt-bmc.h
Applying io_quotes_use            to linux/tps6594_pfsm.h
Applying io_quotes_use            to linux/cxl_mem.h
Applying io_quotes_use            to linux/wmi.h
Applying io_quotes_use            to linux/auto_fs.h
Applying io_quotes_use            to linux/mmtimer.h
Applying io_quotes_use            to linux/f2fs.h
Applying io_quotes_use            to linux/vhost.h
---
Applying io_quotes_use            to sound/asound.h
Applying io_quotes_use            to sound/compress_offload.h
Applying hpux8_bogus_inlines      to math.h
Applying pthread_incomplete_struct_argument to pthread.h
Fixed:  pthread.h
Applying io_quotes_use            to misc/mrvl_cn10k_dpi.h
Applying io_quotes_use            to misc/ocxl.h
Applying io_quotes_use            to misc/cxl.h
Applying io_quotes_use            to misc/xilinx_sdfec.h
Applying io_quotes_def            to unicode/platform.h
Applying sun_malloc               to malloc.h
Applying io_quotes_use            to scsi/cxlflash_ioctl.h
---
Applying machine_name             to x86_64-linux-gnu/bits/unistd_ext.h
Applying io_quotes_use            to x86_64-linux-gnu/asm/mtrr.h
Applying io_quotes_use            to x86_64-linux-gnu/asm/amd_hsmp.h
Applying machine_name             to openssl/e_os2.h
Applying io_quotes_use            to drm/xe_drm.h
Applying io_quotes_use            to drm/radeon_drm.h
Applying io_quotes_use            to drm/panfrost_drm.h
Applying io_quotes_use            to drm/etnaviv_drm.h
Applying io_quotes_use            to drm/lima_drm.h
Applying io_quotes_use            to drm/qaic_accel.h
Applying io_quotes_use            to drm/vc4_drm.h
Applying io_quotes_use            to drm/i915_drm.h
Applying io_quotes_use            to drm/omap_drm.h
Applying io_quotes_use            to drm/pvr_drm.h
Applying io_quotes_use            to drm/amdgpu_drm.h
Applying io_quotes_use            to drm/vgem_drm.h
Applying io_quotes_use            to drm/msm_drm.h
Applying io_quotes_use            to drm/v3d_drm.h
Applying io_quotes_use            to drm/exynos_drm.h
Applying io_quotes_use            to drm/nouveau_drm.h
Applying io_quotes_use            to drm/drm.h
Applying io_quotes_use            to drm/habanalabs_accel.h
Applying io_quotes_use            to drm/tegra_drm.h
Applying io_quotes_use            to rdma/rdma_user_ioctl.h
cc1: note: self-tests are not enabled in this build
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/c++tools/server.cc: In function ‘void server(bool, int, module_resolver*)’:
/checkout/obj/build/x86_64-unknown-linux-gnu/gcc/src/c++tools/server.cc:620:10: warning: ignoring return value of ‘int pipe(int*)’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
---
test [pretty] tests/pretty/import-renames.rs ... ok
test [pretty] tests/pretty/issue-25031.rs ... ok
test [pretty] tests/pretty/let.rs ... ok
test [pretty] tests/pretty/issue-19077.rs ... ok
2025-04-23T05:52:55.472811Z ERROR compiletest::runtest: fatal error, panic: "pretty-printed source does not match expected source\nexpected:\n------------------------------------------\n#[prelude_import]\nuse ::std::prelude::rust_2015::*;\n#[macro_use]\nextern crate std;\n//@ pretty-compare-only\n//@ pretty-mode:hir,typed\n//@ pp-exact:issue-4264.pp\n\n// #4264 fixed-length vector types\n\nfn foo(_: [i32; (3 as usize)]) ({ } as ())\n\nfn bar() ({\n    const FOO: usize = ((5 as usize) - (4 as usize) as usize);\n    let _: [(); (FOO as usize)] = ([(() as ())] as [(); 1]);\n\n    let _: [(); (1 as usize)] = ([(() as ())] as [(); 1]);\n\n    let _ =\n        (((&([(1 as i32), (2 as i32), (3 as i32)] as [i32; 3]) as &[i32; 3])\n                as *const _ as *const [i32; 3]) as *const [i32; (3 as usize)]\n            as *const [i32; 3]);\n\n\n\n\n\n\n\n\n\n    ((::alloc::__export::must_use as\n            fn(String) -> String {must_use::<String>})(({\n            let res =\n                ((::alloc::fmt::format as\n                        for<'a> fn(Arguments<'a>) -> String {format})(((format_arguments::new_const\n                            as\n                            fn(&[&'static str; 1]) -> Arguments<'_> {Arguments::<'_>::new_const::<1>})((&([(\"test\"\n                                        as &str)] as [&str; 1]) as &[&str; 1])) as Arguments<'_>))\n                    as String);\n            (res as String)\n        } as String)) as String);\n} as ())\ntype Foo = [i32; (3 as usize)];\nstruct Bar {\n    x: [i32; (3 as usize)],\n}\nstruct TupleBar([i32; (4 as usize)]);\nenum Baz { BazVariant([i32; (5 as usize)]), }\nfn id<T>(x: T) -> T ({ (x as T) } as T)\nfn use_id() ({\n    let _ =\n        ((id::<[i32; (3 as usize)]> as\n                fn([i32; 3]) -> [i32; 3] {id::<[i32; 3]>})(([(1 as i32),\n                    (2 as i32), (3 as i32)] as [i32; 3])) as [i32; 3]);\n} as ())\nfn main() ({ } as ())\n\n------------------------------------------\nactual:\n------------------------------------------\n#[prelude_import]\nuse ::std::prelude::rust_2015::*;\n#[macro_use]\nextern crate std;\n//@ pretty-compare-only\n//@ pretty-mode:hir,typed\n//@ pp-exact:issue-4264.pp\n\n// #4264 fixed-length vector types\n\nfn foo(_: [i32; (3 as usize)]) ({ } as ())\n\nfn bar() ({\n    const FOO: usize = ((5 as usize) - (4 as usize) as usize);\n    let _: [(); (FOO as usize)] = ([(() as ())] as [(); 1]);\n\n    let _: [(); (1 as usize)] = ([(() as ())] as [(); 1]);\n\n    let _ =\n        (((&([(1 as i32), (2 as i32), (3 as i32)] as [i32; 3]) as &[i32; 3])\n                as *const _ as *const [i32; 3]) as *const [i32; (3 as usize)]\n            as *const [i32; 3]);\n\n\n\n\n\n\n\n\n\n    ((::alloc::__export::must_use as\n            fn(String) -> String {must_use::<String>})(({\n            ((::alloc::fmt::format as\n                    for<'a> fn(Arguments<'a>) -> String {format})(((format_arguments::new_const\n                        as\n                        fn(&[&'static str; 1]) -> Arguments<'_> {Arguments::<'_>::new_const::<1>})((&([(\"test\"\n                                    as &str)] as [&str; 1]) as &[&str; 1])) as Arguments<'_>))\n                as String)\n        } as String)) as String);\n} as ())\ntype Foo = [i32; (3 as usize)];\nstruct Bar {\n    x: [i32; (3 as usize)],\n}\nstruct TupleBar([i32; (4 as usize)]);\nenum Baz { BazVariant([i32; (5 as usize)]), }\nfn id<T>(x: T) -> T ({ (x as T) } as T)\nfn use_id() ({\n    let _ =\n        ((id::<[i32; (3 as usize)]> as\n                fn([i32; 3]) -> [i32; 3] {id::<[i32; 3]>})(([(1 as i32),\n                    (2 as i32), (3 as i32)] as [i32; 3])) as [i32; 3]);\n} as ())\nfn main() ({ } as ())\n\n------------------------------------------\ndiff:\n------------------------------------------\n31\t\n32\t    ((::alloc::__export::must_use as\n33\t            fn(String) -> String {must_use::<String>})(({\n-\t            let res =\n-\t                ((::alloc::fmt::format as\n-\t                        for<'a> fn(Arguments<'a>) -> String {format})(((format_arguments::new_const\n-\t                            as\n-\t                            fn(&[&'static str; 1]) -> Arguments<'_> {Arguments::<'_>::new_const::<1>})((&([(\"test\"\n-\t                                        as &str)] as [&str; 1]) as &[&str; 1])) as Arguments<'_>))\n-\t                    as String);\n-\t            (res as String)\n+\t            ((::alloc::fmt::format as\n+\t                    for<'a> fn(Arguments<'a>) -> String {format})(((format_arguments::new_const\n+\t                        as\n+\t                        fn(&[&'static str; 1]) -> Arguments<'_> {Arguments::<'_>::new_const::<1>})((&([(\"test\"\n+\t                                    as &str)] as [&str; 1]) as &[&str; 1])) as Arguments<'_>))\n+\t                as String)\n42\t        } as String)) as String);\n43\t} as ())\n44\ttype Foo = [i32; (3 as usize)];\n\n\n"
test [pretty] tests/pretty/issue-4264.rs ... FAILED
test [pretty] tests/pretty/issue-68710-field-attr-proc-mac-lost.rs ... ok
test [pretty] tests/pretty/issue-31073.rs ... ok
test [pretty] tests/pretty/lifetime.rs ... ok
test [pretty] tests/pretty/issue-12590-b.rs ... ok
---
------------------------------------------
#[prelude_import]
use ::std::prelude::rust_2015::*;
#[macro_use]
extern crate std;
//@ pretty-compare-only
//@ pretty-mode:hir,typed
//@ pp-exact:issue-4264.pp

// #4264 fixed-length vector types

fn foo(_: [i32; (3 as usize)]) ({ } as ())

fn bar() ({
    const FOO: usize = ((5 as usize) - (4 as usize) as usize);
    let _: [(); (FOO as usize)] = ([(() as ())] as [(); 1]);

    let _: [(); (1 as usize)] = ([(() as ())] as [(); 1]);

    let _ =
        (((&([(1 as i32), (2 as i32), (3 as i32)] as [i32; 3]) as &[i32; 3])
                as *const _ as *const [i32; 3]) as *const [i32; (3 as usize)]
            as *const [i32; 3]);









    ((::alloc::__export::must_use as
            fn(String) -> String {must_use::<String>})(({
            let res =
                ((::alloc::fmt::format as
                        for<'a> fn(Arguments<'a>) -> String {format})(((format_arguments::new_const
                            as
                            fn(&[&'static str; 1]) -> Arguments<'_> {Arguments::<'_>::new_const::<1>})((&([("test"
                                        as &str)] as [&str; 1]) as &[&str; 1])) as Arguments<'_>))
                    as String);
            (res as String)
        } as String)) as String);
} as ())
type Foo = [i32; (3 as usize)];
struct Bar {
    x: [i32; (3 as usize)],
}
struct TupleBar([i32; (4 as usize)]);
enum Baz { BazVariant([i32; (5 as usize)]), }
fn id<T>(x: T) -> T ({ (x as T) } as T)
fn use_id() ({
    let _ =
        ((id::<[i32; (3 as usize)]> as
                fn([i32; 3]) -> [i32; 3] {id::<[i32; 3]>})(([(1 as i32),
                    (2 as i32), (3 as i32)] as [i32; 3])) as [i32; 3]);
} as ())
fn main() ({ } as ())

------------------------------------------
actual:
------------------------------------------
#[prelude_import]
use ::std::prelude::rust_2015::*;
#[macro_use]
extern crate std;
//@ pretty-compare-only
//@ pretty-mode:hir,typed
//@ pp-exact:issue-4264.pp

// #4264 fixed-length vector types

fn foo(_: [i32; (3 as usize)]) ({ } as ())

fn bar() ({
    const FOO: usize = ((5 as usize) - (4 as usize) as usize);
    let _: [(); (FOO as usize)] = ([(() as ())] as [(); 1]);

    let _: [(); (1 as usize)] = ([(() as ())] as [(); 1]);

    let _ =
        (((&([(1 as i32), (2 as i32), (3 as i32)] as [i32; 3]) as &[i32; 3])
                as *const _ as *const [i32; 3]) as *const [i32; (3 as usize)]
            as *const [i32; 3]);









    ((::alloc::__export::must_use as
            fn(String) -> String {must_use::<String>})(({
            ((::alloc::fmt::format as
                    for<'a> fn(Arguments<'a>) -> String {format})(((format_arguments::new_const
                        as
                        fn(&[&'static str; 1]) -> Arguments<'_> {Arguments::<'_>::new_const::<1>})((&([("test"
                                    as &str)] as [&str; 1]) as &[&str; 1])) as Arguments<'_>))
                as String)
        } as String)) as String);
} as ())
type Foo = [i32; (3 as usize)];
struct Bar {
    x: [i32; (3 as usize)],
}
struct TupleBar([i32; (4 as usize)]);
enum Baz { BazVariant([i32; (5 as usize)]), }
fn id<T>(x: T) -> T ({ (x as T) } as T)
fn use_id() ({
    let _ =
        ((id::<[i32; (3 as usize)]> as
                fn([i32; 3]) -> [i32; 3] {id::<[i32; 3]>})(([(1 as i32),
                    (2 as i32), (3 as i32)] as [i32; 3])) as [i32; 3]);
} as ())
fn main() ({ } as ())

------------------------------------------
diff:
------------------------------------------
31 
32     ((::alloc::__export::must_use as
33             fn(String) -> String {must_use::<String>})(({
-             let res =
-                 ((::alloc::fmt::format as
-                         for<'a> fn(Arguments<'a>) -> String {format})(((format_arguments::new_const
-                             as
-                             fn(&[&'static str; 1]) -> Arguments<'_> {Arguments::<'_>::new_const::<1>})((&([("test"
-                                         as &str)] as [&str; 1]) as &[&str; 1])) as Arguments<'_>))
-                     as String);
-             (res as String)
+             ((::alloc::fmt::format as
+                     for<'a> fn(Arguments<'a>) -> String {format})(((format_arguments::new_const
+                         as
+                         fn(&[&'static str; 1]) -> Arguments<'_> {Arguments::<'_>::new_const::<1>})((&([("test"
+                                     as &str)] as [&str; 1]) as &[&str; 1])) as Arguments<'_>))
+                 as String)
42         } as String)) as String);
43 } as ())
44 type Foo = [i32; (3 as usize)];




thread '[pretty] tests/pretty/issue-4264.rs' panicked at src/tools/compiletest/src/runtest.rs:453:18:
```

</details>

--- a/library/alloc/src/macros.rs
+++ b/library/alloc/src/macros.rs
@@ -105,8 +105,7 @@ macro_rules! vec {
 macro_rules! format {
     ($($arg:tt)*) => {
         $crate::__export::must_use({
-            let res = $crate::fmt::format($crate::__export::format_args!($($arg)*));
-            res
+            $crate::fmt::format($crate::__export::format_args!($($arg)*))
         })
     }
 }

--- a/tests/pretty/issue-4264.pp
+++ b/tests/pretty/issue-4264.pp
@@ -31,14 +31,12 @@ fn bar() ({
 
     ((::alloc::__export::must_use as
             fn(String) -> String {must_use::<String>})(({
-            let res =
-                ((::alloc::fmt::format as
-                        for<'a> fn(Arguments<'a>) -> String {format})(((format_arguments::new_const
-                            as
-                            fn(&[&'static str; 1]) -> Arguments<'_> {Arguments::<'_>::new_const::<1>})((&([("test"
-                                        as &str)] as [&str; 1]) as &[&str; 1])) as Arguments<'_>))
-                    as String);
-            (res as String)
+            ((::alloc::fmt::format as
+                    for<'a> fn(Arguments<'a>) -> String {format})(((format_arguments::new_const
+                        as
+                        fn(&[&'static str; 1]) -> Arguments<'_> {Arguments::<'_>::new_const::<1>})((&([("test"
+                                    as &str)] as [&str; 1]) as &[&str; 1])) as Arguments<'_>))
+                as String)
         } as String)) as String);
 } as ())
 type Foo = [i32; (3 as usize)];

